shell/platform/common/client_wrapper/testing/test_codec_extensions.h

```diff
@@ -32,7 +32,7 @@ class Point {
 // variable-length type that includes types handled by the core standard codec.
 class SomeData {
  public:
-  SomeData(const std::string label, const std::vector<uint8_t>& data)
+  SomeData(const std::string& label, const std::vector<uint8_t>& data)
```

Comment:
Could we use `std::move` on line 36 as well here?

Reply:
Ditto above.

Reply:
We don't need to micro-optimize test code performance.